### ISSUE TITLE
Julia language support

### ISSUE THREAD

**Issue by hossein-pourbozorg** [gitpod-io/gitpod#812]
Please, support [Julia Language](https://julialang.org/).
It has a [docker image](https://hub.docker.com/_/julia) and a [VS code extension](https://marketplace.visualstudio.com/items?itemName=julialang.language-julia). So, it can be done with a gitpod config like:
```
image: julia

vscode:
  extensions:
    - julialang.language-julia@0.12.3:lgRyBd8rjwUpMGG0C5GAig==
```
but official support is very better.

**Comment by svenefftinge** [comment]
"Official support" would mean that we have a small guide and an example for Julia.
I'd like to avoid adding Julia support to the standard dev environment as it will further increase the size of the workspace-full image as well as adding more extensions that many will not use.

**Comment by JesterOrNot** [comment]
I could make a Dockerfile

**Comment by JesterOrNot** [comment]
I'll get a repo ready in case it's wanted

**Comment by JesterOrNot** [comment]
https://github.com/JesterOrNot/Gitpod-Julia
incase anyone is interested

**Comment by JesterOrNot** [comment]
first error ERROR: could not load library "/usr/bin/../lib/julia/sys.so"
/usr/bin/../lib/julia/sys.so: cannot open shared object file: No such file or directory

**Comment by jankeromnes** [comment]
@JesterOrNot Super cool! Many thanks for setting this up. I don't even see the error you've mentioned, it just works: 😄

```
$ julia --version
julia version 1.0.1
```

However, I noticed a few things that could be improved in the Dockerfile. I'll make a PR.

**Comment by JesterOrNot** [comment]
thanks! the problem is when starting the repl try just running julia

**Comment by jankeromnes** [comment]
> thanks! the problem is when starting the repl try just running julia

Aha, indeed. The problem is that `sys.so` is installed as `/usr/lib/sys.so` and not `/usr/lib/julia/sys.so`.

But maybe it's simpler/cleaner to just add the extracted directory to the `$PATH`, without moving everything into the system? I think it would work, given that the error shows that Julia may be using relative paths.

**Comment by JesterOrNot** [comment]
I'm sorry could you remind me how that would look?

**Comment by jankeromnes** [comment]
@JesterOrNot Ah, sorry, I even prepared a Pull Request that fixes the problem, but forgot to send it. Sending it now. 🙂

**Comment by svenefftinge** [comment]
@jankeromnes would be great to add a brief section on Julia to our languages & frameworks section pointing to that example repo. @JesterOrNot we can fork it if you prefer or directly point to your example.

**Comment by jankeromnes** [comment]
@svenefftinge Agreed! 😄 https://github.com/gitpod-io/website/pull/260#issuecomment-536486937

**Comment by JesterOrNot** [comment]
what do you mean?

**Comment by jankeromnes** [comment]
@JesterOrNot We have a new "Languages & Frameworks" section in the Gitpod documentation: https://www.gitpod.io/docs/languages_and_frameworks/

Sven is suggesting that we add a brief page about the Julia language, and link to your demo repository as an example of how one can add Julia support to their repository. 🙂 Many thanks for making it!

**Comment by JesterOrNot** [comment]
Could you mention me directly?

**Comment by JesterOrNot** [comment]
Should we pull request it into https://github.com/gitpod-io/workspace-images

**Comment by svenefftinge** [comment]
@JesterOrNot A dedicated docker file for Julia?

**Comment by svenefftinge** [comment]
Oh, I see there is https://github.com/gitpod-io/website/pull/261 already. :-)

**Comment by JesterOrNot** [comment]
Fair enough can this issue be closed so that further conversations of this issue can take place in the pull request?

**Comment by JesterOrNot** [comment]
![image](https://user-images.githubusercontent.com/50386709/67606289-6ec9c880-f746-11e9-9604-5c3a8b1f0be8.png)
